### PR TITLE
Docs 323 mist quickstarts

### PR DESCRIPTION
**Pull request by reidstidolph**
This adds an complete quickstart flow, beginning with SSR120 or SSR130 device pages, and ending with a working standalone device config with LAN -> WAN routing.

Device entry points are unlisted in sidebars, intending to be linked from the Juniper.net docs quickstart page.
- /docs/intro_wa_quickstart_SSR120_onboarding
- /docs/intro_wa_quickstart_SSR130_onboarding

Each one will feed into the workflow.

### CODE REVIEW

File: docs/intro_wa_quickstart_3_templates.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+### Access Policy
+
+Finally you will tie your networks and applications together in an access policy.
+
+![Add access](/img/intro_wa_quickstart_12.gif)
+
+1. Scroll to the access policy section of the template, and select "Add Policy."
+2. Enter a name the name column, and **click the check** to the right of your entry.
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_3_templates.md:76]: "enter a name in the column" ?

File: docs/intro_wa_quickstart_4_siteassign.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Add Device to Site
+sidebar_label: Add Device to Site
+---
+
+At this point you have the following:
+* your SSR device in your inventory.
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_4_siteassign.md:7]: Capitalize first letter in each bulleted list

File: docs/intro_wa_quickstart_SSR120_onboarding.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+sidebar_label: SSR120 Device Onboarding
+---
+
+Congratulations on your new Session Smart Routing (SSR) WAN Edge device!
+
+Let's get it set up in the Mist cloud with WAN Assurance.
+
+## Connect Your Device to the Cloud
+
+<img src="/img/intro_wa_ssr120_quickstart_1.png" alt="Device Connections" width="500"/>
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR120_onboarding.md]: why not use the standard way to add an image to markdown?

File: docs/intro_wa_quickstart_SSR120_onboarding.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Your SSR device uses port 0 (`ge-0`) as a default WAN port to contact Mist for zero-touch provisioning (ZTP). Also you will set up port 3 (`ge-3`) with a LAN network.
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR120_onboarding.md]: is't it `ge-0-0-0` when defined in the mist data model. It should be consistent. Same for `ge-3`

File: docs/intro_wa_quickstart_SSR120_onboarding.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+3. **Power on the device**.
+
+Great job! Your SSR device is now connected to the Mist cloud and awaiting further instructions.
+
+## Claim Your Device
+
+To add the device to your organization WAN Edge inventory, locate the claim label found on the device.
+
+<img src="/img/intro_wa_ssr120_quickstart_2.png" alt="Device Connections" width="500"/>
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR120_onboarding.md]: same as above

File: docs/intro_wa_quickstart_SSR120_onboarding.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+![Claim device](/img/intro_wa_quickstart_claim.gif)
+
+### Mobile App QR Scan
+
+1. Open your MistAI app.
+2. Select "Claim Devices to Org".
+3. Scan the QR code.
+
+<img src="/img/intro_wa_quickstart_mobile_app.png" alt="MistAI app" width="250"/>
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR120_onboarding.md]: same as above

File: docs/intro_wa_quickstart_SSR130_onboarding.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+sidebar_label: SSR130 Device Onboarding
+---
+
+Congratulations on your new Session Smart Routing (SSR) WAN Edge device!
+
+Let's get it set up in the Mist cloud with WAN Assurance.
+
+## Connect Your Device to the Cloud
+
+<img src="/img/intro_wa_ssr130_quickstart_1.png" alt="Device Connections" width="500"/>
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR130_onboarding.md]: why not use standard markdown image syntax?

File: docs/intro_wa_quickstart_SSR130_onboarding.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+Your SSR device uses port 0 (`ge-0`) as a default WAN port to contact Mist for zero-touch provisioning (ZTP). Also you will set up port 3 (`ge-3`) with a LAN network.
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR130_onboarding.md]: same comment as SSR120 page

File: docs/intro_wa_quickstart_SSR130_onboarding.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+3. **Power on the device**.
+
+Great job! Your SSR device is now connected to the Mist cloud and awaiting further instructions.
+
+## Claim Your Device
+
+To add the device to your organization WAN Edge inventory, locate the claim label found on the device.
+
+<img src="/img/intro_wa_ssr130_quickstart_2.png" alt="Device Connections" width="500"/>
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR130_onboarding.md]: same

File: docs/intro_wa_quickstart_SSR130_onboarding.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+![Claim device](/img/intro_wa_quickstart_claim.gif)
+
+### Mobile App QR Scan
+
+1. Open your MistAI app.
+2. Select "Claim Devices to Org".
+3. Scan the QR code.
+
+<img src="/img/intro_wa_quickstart_mobile_app.png" alt="MistAI app" width="250"/>
```
Comment (MichaelBaj) [inline docs/intro_wa_quickstart_SSR130_onboarding.md]: same

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/intro_wa_quickstart_3_templates.md (hunk 1 above)
Comment (reidstidolph) [reply docs/intro_wa_quickstart_3_templates.md:76]: hmmm...yeah the column is called "name", and there are multiple columns so I need to call it out. Let me try to re-word that.

File: docs/intro_wa_quickstart_SSR120_onboarding.md (hunk 3 above)
Comment (reidstidolph) [reply docs/intro_wa_quickstart_SSR120_onboarding.md]: The issue I was having was that it rendered the images stretched the width of the viewport, and they were far too large. I spent some time trying to figure out how to indicate some better width formatting in markdown...and ultimately everywhere I looked people seemed to just say "use inline HTML".

File: docs/intro_wa_quickstart_SSR120_onboarding.md (hunk 4 above)
Comment (reidstidolph) [reply docs/intro_wa_quickstart_SSR120_onboarding.md]: Yeah, good call.

File: docs/intro_wa_quickstart_4_siteassign.md (hunk 2 above)
Comment (reidstidolph) [reply docs/intro_wa_quickstart_4_siteassign.md]: Fixed.

**Review by MichaelBaj — DISMISSED** [review]

File: docs/intro_wa_quickstart_4_siteassign.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+5. Select "Assign to Site".
+
+## Congratulations!
+Mist is now sending template-driven configuration down to your device. After a few minutes to process and apply the configuration, it will begin forwarding sessions from LAN to WAN as described by your policy.
+
+Head over to the WAN Edges menu on the side bar, select your device, and watch events as the device completes ZTP.
+
+As your client devices connected to the LAN are assigned addresses from the WAN Edge DHCP server and begin sending sessions, telemetry will populate the insights page, and Marvis will start analyzing it on your behalf.
+
+Continue reading in the docs for more advanced design use cases, deployment and onboarding scenarios, and AI operations with Mist WAN Assurance!
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_wa_quickstart_4_siteassign.md:28]:
```suggestion
Continue reading in the Mist documentation for more advanced design use cases, deployment and onboarding scenarios, and AI operations with Mist WAN Assurance!
```
Not sure if you are referring to the Mist or the SSR documentation.

File: docs/wan_assurance_ssr130_quickstart.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+## Connect Your Device to the Cloud
+
+<img src="/img/intro_wa_ssr130_quickstart_1.png" alt="Device Connections" width="500"/>
+
+Your SSR device uses port 0 (`ge-0/0/0`) as a default WAN port to contact Mist for zero-touch provisioning (ZTP). Also you will set up port 3 (`ge-0/0/3`) with a LAN network.
+
+1. **Connect port 0** to an Ethernet WAN link capable of providing the device with:
+    * DHCP address assignment
+    * Connectivity Internet and Mist
```
Comment (Chr1st0ph3rTurn3r) [inline docs/wan_assurance_ssr130_quickstart.md:18]:
```suggestion
    * Connectivity to the Internet and Mist
```

File: docs/wan_assurance_ssr130_quickstart.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+Great job! Your SSR device is now connected to the Mist cloud and awaiting further instructions.
+
+## Claim Your Device
+
+To add the device to your organization WAN Edge inventory, locate the claim label found on the device.
+
+<img src="/img/intro_wa_ssr130_quickstart_2.png" alt="Device Connections" width="500"/>
+
+To enter the device claim information into Mist, you can:
+* enter the claim code in Mist
```
Comment (Chr1st0ph3rTurn3r) [inline docs/wan_assurance_ssr130_quickstart.md:36]:
```suggestion
* Enter the claim code in Mist
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Pull request merged**